.travis.yml

```diff
@@ -7,7 +7,7 @@ env:
 
 go:
   - "1.10.x"
-  - "1.11.1"
+  - "1.11.2"
```

Comment:
could you update the README's [requirement](https://github.com/kubernetes-sigs/aws-ebs-csi-driver#requirements-1) section with this change